acceptance_tests/features/steps/reply_to_message_internal.py

```diff
@@ -16,12 +21,51 @@ def internal_user_receive_message(_):
     signed_in_internal(None)
 
 
+@given('the internal user has a conversation in their inbox')
+def create_conversation_internal(_):
+    create_message_external_to_internal()
+    signed_in_internal(None)
+    inbox_internal.go_to()
+    go_to_thread()
+    create_message_internal.enter_text_in_message_body('Body')
+    create_message_internal.click_message_send_button()
+
+
+@given('the internal user has a closed conversation in their inbox')
+def create_and_close_conversation(_):
```

Comment:
This seems to be duplicated with `create_message_external_to_internal` in the `messages_controller.py`. The only difference is one creating an external message and the other creating an internal message

Reply:
Yeah adding them to common makes sense, could add common messages methods into the `messages_controller.py` thats already there